src/view/com/profile/ProfileHeader.tsx

```diff
@@ -218,7 +218,7 @@ const ProfileHeaderLoaded = observer(
             onRefreshAll()
             Toast.show('Account unblocked')
           } catch (e: any) {
-            store.log.error('Failed to block unaccount', e)
```

Comment:
There didn't seem to be a reason for this...

Reply:
I think the replacement should be `Failed to unblock account`.

Reply:
https://github.com/bluesky-social/social-app/pull/810